tracpro/orgs_ext/forms.py

```diff
@@ -162,23 +161,16 @@ def save(self, *args, **kwargs):
 
         instance = super(OrgExtForm, self).save(*args, **kwargs)
 
```

Comment:
I was getting the opposite behavior from what was expected when you update the orgs `how_to_handle_sameday_responses`.  I was able to add instance.save() and get the expected result, but I suspect that you could save in the if statement above (line 152).  Though, I'm not totally sure of the right convention here.